src/components/MultipleAvatars.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Image, Text, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // Array of avatar URL
+    avatars: PropTypes.arrayOf(PropTypes.any),
+
+    // Whether this option is currently in focus so we can modify its style
+    optionIsFocused: PropTypes.bool,
+};
+
+const defaultProps = {
+    avatars: [],
+    optionIsFocused: false,
+};
+
+const MultipleAvatars = (props) => {
+    const {avatars, optionIsFocused} = props;
```

Comment:
Do you mean I don't need to make a component (MultipleAvatars)?

Reply:
No, I mean that we don't do object destructuring of props. So instead of this:

```javascript
const {avatars, optionIsFocused} = props;
```

we would just have `props.avatars` and `props.optionIsFocused`  throughout the rest of the render function. It's just a style thing that we've been consistent about 😁

Reply:
Ok. I got it. 👍